frontend/src/components/Header.tsx

```diff
@@ -54,18 +56,49 @@ const NavLinks = ({
     ];
   }, [t, path, getSearchLink]);
 
+  const getCurrentNavItemIndex = useCallback(
+    (currentPath: string): number => {
+      // handle base case of home page separately
+      if (currentPath.match(homeRegexp)) {
+        return 0;
+      }
+      const index = navLinkList.slice(1).findIndex(({ href }) => {
```

Comment:
some complexity here when we're using the actual list of hrefs - need to remove the homepage since that is handled separately, and need to split out the "?refresh=true" on the search path. That second one we can hope to resolve another way soon, so I'm ok with this!